### ISSUE TITLE
Project Tutorial Proposal - Your Name Here

### ISSUE THREAD

**Issue by SparshRastogi** [codedex-io/projects#24]
1. Visualize Covid-19 Data - Intermediate 
2. Extractive TextSummarizer - Intermediate 
3. Automatic Email Sender - Intermediate 

I would like to work on any of these projects

**Comment by Bobliuuu** [comment]
Hey @SparshRastogi, thanks for reaching out! All your ideas are very interesting 💯 

We'd love for you to do the text summarizer! 

Can you submit the code first before a draft?

Also, can you send us your name and one-line about yourself to [team@codedex.io](mailto:team@codedex.io)? Thanks!

**Comment by SparshRastogi** [comment]
Sure,just to confirm by code before draft you mean the code of .py file instead of creating the whole directory as in template right?

**Comment by Bobliuuu** [comment]
Yes, please send us the .py file before you write the project tutorial!

**Comment by SparshRastogi** [comment]
@Bobliuuu I have uploaded the file in #26 pull request can you please review it

**Comment by Bobliuuu** [comment]
Hey @SparshRastogi reviewed! Once you make the changes, you may start your project tutorial!
